src/integrationTest/resources/application.yaml

```diff
@@ -60,6 +60,7 @@ idam:
 
 authorised.services: lau_frontend
 authorised.roles: cft-audit-investigator
+db.allow.delete.record: true
```

Comment:
Can you add the environment variable DB_ALLOW_DELETE_RECORD here as well, thank.

Reply:
done